packages/next/export/worker.js

```diff
@@ -73,14 +74,18 @@ process.on(
           ...headerMocks
         }
 
+        if (sprPage && isDynamicRoute(page)) {
+          // pass via `req` to avoid adding code to serverless bundle
+          req.skipGIP = true
```

Comment:
I'd rather this append `skeleton` to the `query` object while pre-rendering.

This is how we do it for AMP, same idea:
```js
req.url += (req.url.includes('?') ? '&' : '?') + 'amp=1'
```

Reply:
That works, maybe something like `nextFallback` instead of `skeleton` 🤔 Although we don't need to append it to `req.url` since we pass in the parsed `query` here